asmcomp/compilenv.ml

```diff
@@ -44,12 +52,14 @@ module CstMap =
 type structured_constants =
   {
     strcst_shared: string CstMap.t;
+    strcst_original: string StringMap.t;
```

Comment:
I'm currently cleaning the diff and that was part of some changes that were removed.